tests/cli/test_main_notices.py

```diff
@@ -383,7 +383,7 @@ def test_notices_does_not_interrupt_command_on_failure(
 
     _, _, exit_code = conda_cli("env", "remove", "--name", env_name)
 
-    assert exit_code is None
+    assert exit_code == 0
```

Comment:
Updating this test to assert the correct exit code; [`conda/cli/main_remove.py`](https://github.com/conda/conda/blob/main/conda/cli/main_remove.py) has been [changed recently](https://github.com/conda/conda/pull/13173) to return 0, so [`conda/cli/main_env_remove.py`](https://github.com/conda/conda/pull/13168/files#diff-5f51f005f7d00bb0e703484e3675626de6348630557da60ec300f77bacb8991cR83) is doing the same thing.